src/components/Review/ReviewSectionRowAssigned.tsx

```diff
@@ -14,84 +13,61 @@ const ReviewSectionRowAssigned: React.FC<ReviewSectionComponentProps> = ({
   const isReviewLocked: boolean =
     action === ReviewAction.canSelfAssignLocked || action === ReviewAction.canContinueLocked
   return (
-    <Grid.Column>
-      <div style={inlineStyle.container}>
-        {isAssignedToCurrentUser && action === ReviewAction.canSelfAssign ? (
-          <Label style={inlineStyle.yourAssignmentLabel} content={strings.LABEL_ASSIGNMENT_SELF} />
-        ) : isSelfAssigned ? (
-          <Label
-            icon={<Icon name="circle" size="tiny" />}
-            style={inlineStyle.yourAssignmentLabel}
-            content={strings.LABEL_ASSIGNED_TO_YOU}
-          />
-        ) : isReviewLocked ? (
-          <Label
-            icon={<Icon name="ban" size="small" />}
-            style={inlineStyle.lockedAssignmentLabel}
-            content={
-              <>
-                {strings.LABEL_ASSIGNMENT_LOCKED}
-                <Reviewer user={assignment.reviewer} isCurrent={isAssignedToCurrentUser} />
-              </>
-            }
-          />
-        ) : (
-          <Label
-            style={inlineStyle.othersAssignmentLabel}
-            content={
-              <>
-                {action === ReviewAction.canSelfAssign
-                  ? strings.LABEL_ASSIGNMENT_AVAILABLE
-                  : strings.LABEL_REVIEWED_BY}
-                <Reviewer user={assignment.reviewer} isCurrent={isAssignedToCurrentUser} />
-              </>
-            }
-          />
-        )}
-      </div>
+    <Grid.Column className="padding-zero">
+      {isAssignedToCurrentUser && action === ReviewAction.canSelfAssign ? (
+        <Label className="simple-label" content={strings.LABEL_ASSIGNMENT_SELF} />
```

Comment:
Just wondering why this needs to be a "Label" component as opposed to just straight text? Maybe the formatting (i.e. vertical alignment) would work better for straight text. Doesn't seem like anything is really being "labelled".

Reply:
Can you give me an example please? I was using like that to keep all the Global layout we have set for Semantic-ui components, and using text, not wrapped with something seemed difficult to set styles, since there is no tags related to it.

Reply:
Yeah okay, I see what you're saying.  "simple-label" works as an extension of "Label". Cool.

Just wondering what's causing the vertical alignment problem though. Looking at the `.simple-label` definition, I think the problem is the "height" specification. It's artificially forcing a height onto something whose height really should be determined by the text itself. If I remove `height: 20px;`, it correctly centers vertically. There's still an issue with the rows being taller than what you want (I think), but it looks better and it's more of an issue of wrangling margins and padding to make it narrower, I'd say.

Reply:
> Looking at the .simple-label definition, I think the problem is the "height" specification

Yes, I noticed that too, and have removed on commit 32c935a